src/main/java/com/google/cloud/tools/maven/StageMojo.java

```diff
@@ -216,15 +215,20 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
     getLog().info("Staging the application to: " + stagingDirectory);
 
-    appengineWebXml =
-        new AppEngineWebXml(new File(sourceDirectory.toString() + "/WEB-INF/appengine-web.xml"));
-
     configureAppEngineDirectory();
 
```

Comment:
feels like you could just resolve appengienWeb.xml here and do an if(exists) to start this loop?

Reply:
unless isStandardStaging is used somewhere else.

Reply:
I use isStandardStaging in the deploy all task, which is why I changed this in the first place.

Reply:
Also in `AbstractSingleYamlDeployMojo`.